app/components/pages/Dashboard/index.js

```diff
@@ -21,6 +21,7 @@ const BoxHiddenFromMobile = styled(Box)`
 
 const DashboardPanel = styled(Tabs.Panel)`
   min-height: 300px;
+  margin-bottom: 12px;
```

Comment:
Oh. Sorry. This isn't actually the right component to add it to. This is adding an extra bottom margin to each dashboard list item.

I wasn't paying attention to the code. I just assumed from the name that `DashboardPanel` meant the entire list.

It should be on this component: https://github.com/elifesciences/elife-xpub/pull/895/files#diff-ddd4166fe5ead236eb34c57ce9d7b2c2R67

Do you see what I mean?